### PR TITLE
[Enchancement](compatible) show dateV2/datetimeV2 to date/datetime 

### PR DESCRIPTION
**Pull request by Cai-Yao**
# Proposed changes

Issue Number: close #xxx

# Problem summary

show dateV2/datetimeV2 to date/datetime 

- modify `show create table`
- modify `desc table`
- use `desc table all` to get real type from column `ColumnType`

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by yiguolei — COMMENTED** [review]
Add regression test please.

**Comment by morningman** [comment]
Do not changing `comment`. The `comment` is "user specified info", you should not modify any content specified by user.
I suggest to extent the column of `desc table all` statement, to add a new column to indicate the v2/v3 info

File: fe/fe-core/src/main/java/org/apache/doris/analysis/DescribeStmt.java [hunk 1]
```diff
@@ -67,6 +67,7 @@ public class DescribeStmt extends ShowStmt {
                     .addColumn(new Column("IndexKeysType", ScalarType.createVarchar(20)))
                     .addColumn(new Column("Field", ScalarType.createVarchar(20)))
                     .addColumn(new Column("Type", ScalarType.createVarchar(20)))
+                    .addColumn(new Column("ColumnType", ScalarType.createVarchar(20)))
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/DescribeStmt.java:70]: InternalType

File: fe/fe-core/src/main/java/org/apache/doris/analysis/DescribeStmt.java [hunk 2]
```diff
@@ -205,6 +211,7 @@ public void analyze(Analyzer analyzer) throws UserException {
                                     "",
                                     column.getName(),
                                     column.getOriginType().toString(),
+                                    column.getOriginType().toString(),
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/DescribeStmt.java:214]: 是不是跟213 行重复了

File: fe/fe-core/src/main/java/org/apache/doris/analysis/DescribeStmt.java [hunk 3]
```diff
@@ -288,6 +301,13 @@ public List<List<String>> getResultRows() throws AnalysisException {
             List<List<String>> rows = node.fetchResult().getRows();
             List<List<String>> res = new ArrayList<>();
             for (List<String> row : rows) {
+                // show DATEV2/DATETIMEV2 to DATE/DATETIME for compatibility
+                if (row.get(1).length() >= 6 && row.get(1).substring(0, 6).equals("DATEV2")) {
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/DescribeStmt.java:305]: why add these code?

File: fe/fe-core/src/main/java/org/apache/doris/analysis/DescribeStmt.java (hunk 2 above)
Comment (Cai-Yao) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/DescribeStmt.java:214]: > 是不是跟213 行重复了
填充type和新加的columntype两个列

**Review by yiguolei — APPROVED** [review]
LGTM

**Comment by yiguolei** [comment]
run buildall

**Pull request merged**